### ISSUE TITLE
Change default RucioInjector.listTiersToInject value

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#9645]
**Impact of the bug**
WMAgent

**Describe the bug**
While we are still not using RucioInjector, that default value:
```
config.RucioInjector.listTiersToInject = ['NANOAOD', 'NANOAODSIM']
```
will actually cause problems in PhEDExInjector.
I wanted to make sure they would not work on the same datatier, the problem is that, if RucioInjector is disabled in the configuration, but still has that default value, those two datatiers will actually be skipped by PhEDExInjector as well.
In short, that default value only makes sense once we start gradually using RucioInjector.

**How to reproduce it**
none

**Expected behavior**
Default `listTiersToInject` to a empty list; and change the configuration once we are ready to switch some data over to RucioInjector.

**Additional context and error message**
none
